src/wolfgame.py

```diff
@@ -6707,24 +6707,30 @@ def listroles(cli, nick, chan, rest):
     elif rest[0] and not rest[0].isdigit():
         gamemode = rest[0]
         if gamemode not in var.GAME_MODES.keys():
-            gamemode, _ = complete_match(rest[0], var.GAME_MODES.keys() - ["roles", "villagergame"] - var.DISABLED_GAMEMODES)
-        validMode = gamemode in var.GAME_MODES.keys() and gamemode != "roles" and gamemode != "villagergame" and gamemode not in var.DISABLED_GAMEMODES
-        if validMode and hasattr(var.GAME_MODES[gamemode][0](), "ROLE_GUIDE"):
-            mode = var.GAME_MODES[gamemode][0]()
+            gamemode = complete_match(rest[0], var.GAME_MODES.keys() - ["roles", "villagergame"] - var.DISABLED_GAMEMODES)
+            if len(gamemode) > 1:
+                msg.append("{0}: {1} is not a valid game mode. Possbile modes are: {2} ".format(nick, rest[0], ", ".join(gamemode)))
+                rest = []
+                roleindex = {}
+            else:
+                gamemode = gamemode.pop()
+        else:
+            validMode = gamemode in var.GAME_MODES.keys() and gamemode != "roles" and gamemode != "villagergame" and gamemode not in var.DISABLED_GAMEMODES
+            if validMode and hasattr(var.GAME_MODES[gamemode][0](), "ROLE_GUIDE"):
+                mode = var.GAME_MODES[gamemode][0]()
             if hasattr(mode, "ROLE_INDEX") and hasattr(mode, "ROLE_GUIDE"):
                 roleindex = mode.ROLE_INDEX
                 roleguide = mode.ROLE_GUIDE
-            elif gamemode == "default" and "ROLE_INDEX" in var.ORIGINAL_SETTINGS and "ROLE_GUIDE" in var.ORIGINAL_SETTINGS:
+            elif gamemode == "default" and "ROLE_INDEX" in var.ORIGINAL_SETTINGS and "ROLE_GUIDE" in var.   ORIGINAL_SETTINGS:
                 roleindex = var.ORIGINAL_SETTINGS["ROLE_INDEX"]
                 roleguide = var.ORIGINAL_SETTINGS["ROLE_GUIDE"]
-            rest.pop(0)
-        else:
-            if validMode and not hasattr(var.GAME_MODES[gamemode][0](), "ROLE_GUIDE"):
-                msg.append("{0}: {1}roles is disabled for the {2} game mode.".format(nick, botconfig.CMD_CHAR, gamemode))
             else:
-                msg.append("{0}: {1} is not a valid game mode.".format(nick, rest[0]))
-            rest = []
-            roleindex = {}
+                if validMode and not hasattr(var.GAME_MODES[gamemode][0](), "ROLE_GUIDE"):
+                    msg.append("{0}: {1}roles is disabled for the {2} game mode.".format(nick, botconfig.CMD_CHAR, gamemode))
+                else:
+                    msg.append("{0}: {1} is not a valid game mode.".format(nick, rest[0]))
+                    rest = []
+                    roleindex = {}
```

Comment:
these 2 lines are further indented compared to previous. Is this intentional and if so, why?